editor/plugins/script_editor_plugin.cpp

```diff
@@ -1924,16 +1924,18 @@ void ScriptEditor::_update_members_overview_visibility() {
 void ScriptEditor::_update_members_overview() {
 	members_overview->clear();
 
-	Node *current = tab_container->get_child(tab_container->get_current_tab());
-	ScriptTextEditor *ste = tab_container->get_child(tab_container->get_current_tab())->cast_to<ScriptTextEditor>();
-	if (!ste) {
-		return;
-	}
+	if(tab_container->get_child_count() > 0) {
```

Comment:
`clang-format` will complain about the style here, see http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

Reply:
Fixed.